staging/src/k8s.io/kube-scheduler/config/v1alpha2/types.go

```diff
@@ -42,8 +42,7 @@ type KubeSchedulerConfiguration struct {
 	// SchedulerName is name of the scheduler, used to select which pods
 	// will be processed by this scheduler, based on pod's "spec.SchedulerName".
 	SchedulerName *string `json:"schedulerName,omitempty"`
-	// AlgorithmSource specifies the scheduler algorithm source.
-	AlgorithmSource SchedulerAlgorithmSource `json:"algorithmSource"`
```

Comment:
Actually, can we add a DEPRECATED notice and TODO in `pkg/scheduler/apis/config` for this field?

Reply:
Added

Reply:
I think you are going to upset golint. You can put the DEPRECATED notice after the main comment.

While you are on it, do `TODO(#87526)`. Which reminds me that this PR description shouldn't say "fixes", but "part of".